packages/autocannon-ui-frontend/src/components/ResultSet.jsx

```diff
@@ -53,7 +53,7 @@ const useStyles = makeStyles(theme => ({
   }
 }))
 
-export default function ResultSet({ data, onChangeSelection }) {
+export default function ResultSet({ data, onChangeSelection, resultIndex = 0 }) {
```

Comment:
don't default it, this property doesn't have a sensible default

Reply:
Alright, removing the default